lib/tasks/dev.rake

```diff
@@ -70,6 +71,30 @@ task({ :sample_data => :environment}) do
     p membership.errors.full_messages
   end
 
+    30.times do
+    membership = Membership.new
+    membership.group_id = Group.all[3].id
+    membership.contact_id = Contact.all.sample.id
+    membership.save
```

Comment:
You can condense these into one line using `create` instead of `new`

```suggestion
    membership = Membership.create(group: Group.all[3].id, contact: Contact.all.sample.id)
```

you can drop the `_id` and directly use the associations `group` and `contact` and Rails will figure it out.